tests/zfs-tests/tests/functional/removal/removal_condense_export.ksh

```diff
@@ -28,9 +28,32 @@ function reset
 	default_cleanup_noexit
 }
 
+function gethz
+{
+	typeset CONFFILE=/boot/config-$(uname -r)
+	typeset CONF
+	typeset HZ=1000
+	typeset CONFIG_HZ
+
+	if ! is_linux; then
+		echo 100
+		return 0
+	fi
+
+	if [[ -f $CONFFILE ]]; then
+		CONF=$(grep "^CONFIG_HZ=[0-9]" $CONFFILE 2> /dev/null)
+		if [[ "$CONF" ]]; then
+			eval $CONF
+			[[ "$CONFIG_HZ" ]] && HZ=$CONFIG_HZ
+		fi
+	fi
+	echo $HZ
+	return 0
+}
+
 default_setup_noexit "$DISKS" "true"
 log_onexit reset
-log_must set_tunable64 zfs_condense_indirect_commit_entry_delay_ticks 1000
+log_must set_tunable64 zfs_condense_indirect_commit_entry_delay_ticks $(gethz)
```

Comment:
Instead why don't we change `zfs_condense_indirect_commit_entry_delay_ticks` to `zfs_condense_indirect_commit_entry_delay_ms`.  As I'm sure you discovered, in user space we don't have a reliably way to check the HZ on the running system on all distributions.

Reply:
I'll change it to a ms-based parameter for the next push.  I was certainly thinking of doing so but wanted to keep compatibility with the upstream work.  In retrospect, I do think it should be changed and is more consistent with the other time-related parameters.

Reply:
FYI, it doesn't really matter if we sleep for the "right" amount of time in userland, since this setting isn't used by ztest.  But I agree that standardizing on ms would probably be better.

Reply:
Yes, I noticed that the userland kernel uses a fixed value.  I'm pretty sure the reason we started fiddling with the value was to get this particular ZTS test to pass.